src/site/content/en/lighthouse-performance/dom-size/index.md

```diff
@@ -13,6 +13,7 @@ cause longer [style calculations](https://developers.google.com/web/fundamentals
 and produce costly [layout reflows](https://developers.google.com/speed/docs/insights/browser-reflow?utm_source=lighthouse&utm_medium=cli). 
 Lighthouse reports the total DOM elements for a page, it's maximum child elements,
 and it's maximum DOM depth in the Diagnostics section:
```

Comment:
look like a performance edit snuck in this branch.

Reply:
I'm checking it out, to hopefully no longer have it in branch. It's been tested and updated in performance branch.